<launch>
  <node name="navigation_node" pkg="robot_slam" type="main.py" output="screen">
      <!--                                0      1      2      3       4      5      6      7         8       9      10    11    12     13    14     15     16  -->
      <param name="goalListX" value="   0.5,    1.69,   2.8,   2.95,   3,    3.1,    2.83,   1.7,   0.46,  0.31 ,   0.31,  0.28,  2.42,  2.75, 1.1,   0.97, 1.77"/>
      <param name="goalListY" value="   -0.3,   -0.3,  -0.25,  -0.32,  -1.42, -2.66,  -2.66,   -3,     -3,   -2.65,   -1.47, -0.4, -2.13,-2.55, -2.37, -0.88,-1.45"/>
      <param name="goalListYaw" value="   90,    90,    90,     0,     0,      0,   -90,     -90,    -90,   -180 ,  -180 , -180,  0,      0,   -180,  -180,  0"/>
  </node>
</launch>
